### PR TITLE
addressing log4j CVEs

### PR DESCRIPTION
**Pull request by deepthi**
Documenting the actions taken in response to the log4j CVEs.

### CODE REVIEW

File: content/en/blog/2021-12-17-log4j-cve-patches.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+description: "Patch releases for log4j CVEs" 
+---
+
+## Background
+A critical vulnerability [CVE-2021-44228](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228) in the Apache Log4j logging library was disclosed on Dec 9. 
+The project provided release 2.15.0 with a patch that mitigates the impact of this CVE. It was quickly found that the initial patch was insufficient, and an additional CVE
+[CVE-2021-45046](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046) followed. This has been fixed in release 2.16.0.
+
+## Who is affected?
+The bulk of vitess code is in golang, and is unaffected by these vulnerabilities. The only component that is affected is the vitess-jdbc driver.
```
Comment (deepthi) [inline content/en/blog/2021-12-17-log4j-cve-patches.md:16]: @harshit-gangal I grepped for `log` under `java` and found it only in the jdbc folder.
I did find an artifact on maven-central called `vitess-parent`. What is that and do we need to mention it here?

File: content/en/blog/2021-12-17-log4j-cve-patches.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+```
+<log4j2.version>2.16.0</log4j2.version>
+```
+You can see an example at https://github.com/vitessio/vitess/pull/9387/files
+
+### If you download vitess artifacts from maven-central
+We have released new artifacts for the supported releases.
+* v10.0.4
+* v11.0.3
+* v12.0.2
```
Comment (deepthi) [inline content/en/blog/2021-12-17-log4j-cve-patches.md]: @askdba @frouioui we should link to the releases once they are ready.

Comment (frouioui) [reply content/en/blog/2021-12-17-log4j-cve-patches.md]: got it

**Review by frouioui — APPROVED** [review]
Looks great to me, thank you @deepthi!

**Comment by frouioui** [comment]
Marking this as `do-not-merge` as we should wait until December 17 to publish this.

**Pull request merged**